pkg/secrets/secrets.go

```diff
@@ -80,3 +85,60 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 
 	return userDataSecret, nil
 }
+
+// Encrypt attempts to encrpyt a plaintext message using the given key as an encoded passphrase
+func Encrypt(plainText string, key []byte) (string, error) {
+	if key == nil {
+		return "", errors.New("encryption passphrase cannot be nil")
+	}
+	// prepare PGP block to hold encrpyted data
+	msgBuffer := bytes.NewBuffer(nil)
+	encoder, _ := armor.Encode(msgBuffer, "ENCRYPTED DATA", nil)
```

Comment:
It's really just a tag that the wraps encrypted data contains
```
-----BEGIN ENCRYPTED DATA-----
base64 encoded data
-----END ENCRYPTED DATA----
```
Since it's not necessary, I'll just remove it and pass in the empty string.